roles/ceph-osd/tasks/scenarios/journal_collocation.yml

```diff
@@ -29,9 +29,11 @@
     - combined_ispartition_results.results
     - devices
   when:
+    not item.0.skipped and
+    not item.1.skipped and
     item.0.rc != 0 and
     item.1.rc != 0 and
     journal_collocation and not
-    osd_auto_discovery
+    osd_auto_discovery and
```

Comment:
Done, sorry. I rearranged this during testing and didn't do a very good job of un-rearranging it.